test/common-utils/test.sh

```diff
@@ -13,6 +13,8 @@ check "zsh" zsh --version
 check "ps" ps --version
 check "Oh My Zsh! theme" test -e $HOME/.oh-my-zsh/custom/themes/devcontainers.zsh-theme
 check "zsh theme symlink" test -e $HOME/.oh-my-zsh/custom/themes/codespaces.zsh-theme
+check "zsh theme filename" test -e $HOME/.oh-my-zsh/custom/themes/codespaces.zsh-theme
```

Comment:
I would take the meaning of -e and symlink to mean xor... not sure @samruddhikhandale